wolfcrypt/src/asn.c

```diff
@@ -24,6 +24,7 @@
     #include <config.h>
 #endif
 
+#include <wolfssl/wolfcrypt/wc_port.h>
```

Comment:
Why is wc_port.h needed here?  Isn't it already included via asn.h -> (includes) types.h -> (includes) wc_port.h?